test/integration/targets/vmware_inventory/aliases

```diff
@@ -0,0 +1,4 @@
+posix/ci/cloud/group1/vcenter
+cloud/vcenter
+skip/python3
```

Comment:
Why is python 3 being skipped?

Reply:
vmware_inventory.py is not supported for Python 3. Right now, we don't have any plan around supporting that.